state_machine_node:
  ros__parameters:
    logging:
      lantern_log_path: "lanterns_log.csv"
      event_log_path: "statemachine_events.log"
    topics:
      topic_state: "statemachine/state"
      topic_cmd_: "statemachine/cmd"
      topic_heartbeat: "heartbeat"

      topic_lantern_detections: "detected_lanterns"

      topic_viz_checkpoint: "statemachine/viz/checkpoint"
      topic_viz_path: "statemachine/viz/path_flight"
    time:
      alive_tol_sec: 10.0
      pub_state_loop_sec: 1.0
    lanterns:
      lantern_merge_dist_m: 0.2
    checkpoints:
      [-38.0, 10.0, 15.0,
       -320.0, 8.0, 17.0,
       38.0, 10.0, 6.6]
    nodes:
      ["lantern_detector_node",
       "path_planner_node",
       "controller_node"]
    
